config/config.go

```diff
@@ -1,16 +1,70 @@
 package config
 
-import "time"
+import (
+	ldbopt "github.com/syndtr/goleveldb/leveldb/opt"
+)
 
-// ReplicaConfig specifies the information needed for a Replica and Version manager to
-// maintain global object versioning and provenance.
-// TODO: this configuration is pulled from a service context not a library context.
+// DefaultConfig is used if the user does not specify a configuration
+var DefaultConfig = Config{
+	Versions: ReplicaConfig{
+		PID:    1,
+		Region: "local",
+		Name:   "localhost",
+	},
+}
+
+// New creates a configuration with the required options and can also be used to specify
+// optional configuration e.g. for engine-specific operations.
+func New(options ...Option) Config {
```

Comment:
The purpose of `DefaultConfig` and `New` with options is so that someone can start a Honu database with no configuration what so ever except for the path to open the database. Advanced usage, e.g. for replication, requires options, but this at least gets out of the way of the user.

Reply:
I like it! This would be a great thing for us to [add to the project README](#17) so that we're letting users know about this feature

Reply:
Thank you for creating an issue for that!